src/main/java/io/fabric8/maven/docker/config/BuildXConfiguration.java

```diff
@@ -42,6 +42,18 @@ public class BuildXConfiguration implements Serializable {
     @Parameter
     private String nodeName;
 
+    /**
+     * Value to be passed to {@code --cache-from} option of BuildX build.
+     */
+    @Parameter
+    private String cacheFrom;
```

Comment:
Is format of `cacheFrom` used in buildx different from one used in legacy docker builds? I see  we already have `cacheFrom` configuration option in `build` > `cacheFrom`. If they both have similar format maybe we can see the existing configuration option.

Reply:
I am not an expert in this area, but from [docs](https://docs.docker.com/engine/reference/commandline/build/#cache-from) of `docker build --cache-from` I understand that the argument can only be a registry image: "flag pointing to an image in the registry". 

While for `docker buildx build --cache-from` there are [multiple different options](https://docs.docker.com/engine/reference/commandline/buildx_build/#cache-from): "Supported types are registry, local, gha and s3". 

So the BuildX flag covers the values from the legacy builder, and more. 

From my point of view as a plugin user, `cacheFrom` and `cacheTo` should go together in case of BuildX usage. We cannot add `cacheTo` to `build` section as legacy Docker build does not support this. Having to define `cacheFrom` in `build`, and `cacheTo` in `build / buildX` is confusing from user standpoint. For this reason, I would suggest to have both `cacheFrom` and `cacheTo` in `buildX`, even though there is some overlap with `build [cacheFrom]`.